feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/staticgueencap_and_bgp_path_selection_test.go

```diff
@@ -0,0 +1,2024 @@
+package staticgueencapbgppathselection_test
```

Comment:
Lets wait for https://github.com/openconfig/featureprofiles/pull/4927 to be approved. We will need to modify the code to change to IPv6 Transport